femmt/examples/get_avg_multiple_files.py

```diff
@@ -0,0 +1,87 @@
+import pandas as pd
```

Comment:
Is this file an example file?
The header and also the file name does not make clear that this is for time-domain simulation

is it possible to include this functionality into the toolbox? does it make sense?